packages/solid-crs-manage/lib/features/loan/components/loan-detail.component.ts

```diff
@@ -22,7 +51,35 @@ export class LoanDetailComponent extends RxLitElement {
   render(): TemplateResult {
 
     return html`
-      LOAN DETAIL
+      <nde-large-card .showImage="${false}" .showHeader="${false}">
+        <div slot="content">
+          <h3>
+            ${ this.translator?.translate('loan.detail.card.title')
+    .replace('{{institution}}', this.loanRequest?.from)}
+          </h3>
```

Comment:
font size of the title seems off

![image](https://user-images.githubusercontent.com/32491056/153561336-a738c351-650e-416b-8bb9-eeec9c2d7a68.png)

is the same size as 'Opmerkingen' in Figma. (around 17px, `--font-size-normal`?)